template-parts/excerpt/excerpt-audio.php

```diff
@@ -9,17 +9,13 @@
  * @since 1.0.0
  */
 
-// Print the 1st instance of an audio block.
-// If there was no audioblock, print the 1st audio-embed block.
-if (
-	! twenty_twenty_one_print_first_instance_of_block( 'core/embed', get_the_content() ) &&
-	! twenty_twenty_one_print_first_instance_of_block( 'core/audio', get_the_content() )
-) {
+$content = get_the_content();
 
-	// Fallback to the content.
-	the_content();
```

Comment:
If there is no `core/audio` or `core/embed`/`core-embed/*` block we'll need to fallback to `the_content()` instead of `the_excerpt()`